c2corg_ui/templates/waypoint/edit.html

```diff
@@ -10,8 +10,8 @@
                                 ng-bind="id ? mainCtrl.page_title('Editing a waypoint') : mainCtrl.page_title('Creating a waypoint')">${show_title('Create/edit waypoint')}</title></%block>\
 <%block name="metarobots"><meta name="robots" content="noindex,nofollow"></%block>\
 
-<h1 ng-init="id=${waypoint_id if waypoint_id else 0}" ng-bind="id ? ('Editing a waypoint' | translate) : ('Creating a waypoint' | translate)" class="text-center"></h1>
-<form app-document-editing="waypoints" app-document-editing-model="waypoint"
+<h1 ng-init="title='${waypoint_id}'" ng-bind="title != 'None' ? ('Editing a waypoint' | translate) : ('Creating a waypoint' | translate)" class="text-center"></h1>
```

Comment:
As in ``c2corg_ui/templates/route/edit.html`` please stick to the "0" solution instead of "None".